### ISSUE TITLE
Remove release wait on content release

### ISSUE THREAD

**Issue by swirtSJW** [department-of-veterans-affairs/va.gov-cms#2081]
## Description
In prod.settings.php we are setting a 5 minute release wait when triggering a content release.
This 5 min wait is no longer needed.   It should be changed to 0

## Acceptance Criteria
* Testable_Outcome_X
* Testable_Outcome_Y
* Testable_Outcome_Z

## Labels
- [/] Issue type (red) (defaults to task)
- [ ] CMS subsystem (green)
- [ ] CMS practice area (blue)
- [ ] CMS objective (orange)
- [ ] CMS-supported product (black)
